.github/actions/move-prover-setup/action.yaml

```diff
@@ -0,0 +1,52 @@
+inputs:
+  GIT_CREDENTIALS:
+    description: "Optional credentials to pass to git"
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - run: sudo apt-get update && sudo apt-get install build-essential ca-certificates clang curl git libpq-dev libssl-dev pkg-config lsof lld --no-install-recommends --assume-yes
+      shell: bash
+
+    - uses: dsherret/rust-toolchain-file@v1
+
+    # rust-cache action will cache ~/.cargo and ./target
+    # https://github.com/Swatinem/rust-cache#cache-details
+    - name: Run cargo cache
+      uses: Swatinem/rust-cache@359a70e43a0bb8a13953b04a90f76428b4959bb6 # pin@v2.2.0
+
+    - name: install protoc and related tools
+      shell: bash
+      run: scripts/dev_setup.sh -b -r -y -P -t
+
+    - name: install prover dependencies
+      shell: bash
+      run: scripts/dev_setup.sh -b -p -y
```

Comment:
can you just combine these `dev_setup.sh` calls? iiuc calling them in succession was causing some weird issues previously

Reply:
done

Reply:
thanks!